lib/dash/content_protection.js

```diff
@@ -203,7 +203,7 @@ shaka.dash.ContentProtection = class {
    * @return {string}
    */
   static getFairPlayLicenseUrl(element) {
-    if (shaka.util.Platform.isMediaKeysPolyfilled()) {
+    if (shaka.util.Platform.isMediaKeysPolyfilled('apple')) {
```

Comment:
It seems weird to me that we have the constants `shaka.polyfill.PatchedMediaKeysWebkit.apiName_` and `shaka.polyfill.PatchedMediaKeysApple.apiName_`, yet only use them once each. And when referencing the values outside of their respective classes, we just use the string literals `'webkit'` and `'apple'`.

I suppose it's worth it to simplify the dependencies...

Reply:
It is to simplify dependencies, and avoid add dependencies in our code to polyfills.